webpack.shared.config.js

```diff
@@ -26,7 +26,8 @@ module.exports = {
         test: /\.(sass|scss)$/,
         include: [
           path.resolve(__dirname, 'source'),
-          path.resolve(__dirname, 'drupal'),
+          path.resolve(__dirname, 'app-drupal'),
+          path.resolve(__dirname, 'app-pl'),
```

Comment:
You forgot `app-grav`, but I'm sure Critter will forgive you. In all seriousness, I think this is great.

Reply:
(@mikecrittenden). If we say Grav three times in a dark bathroom, staring into a mirror, he appears behind you with opinions about markdown-based, database-less CMS.

Reply:
IT'S THE DX OF A SSG WITH THE POWER OF A CMS